pkg/kademlia/routinggraph/routinggraph.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package routinggraph
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/pb"
+)
+
+func extendPrefix(prefix string, bit bool) string {
```

Comment:
Move this as last func.

More important funcs first, utilties as last. It helps to find the important things in the package/file faster.